src/framework/components/camera/camera_system.js

```diff
@@ -46,6 +46,11 @@ pc.extend(pc, function () {
             type: "boolean",
             defaultValue: true
         }, {
+            name: "frustumCulling",
+            displayName: "Frustum culling",
+            type: "boolean",
+            defaultValue: false
+        }, {
```

Comment:
Let's not expose new stuff in the UI because they will need migrations. Instead can we make a list of what's been added so we can add everything in one go?